sharing_portal/static/sharing_portal/artifacts.css

```diff
@@ -247,3 +248,15 @@
 .artifactVersion__doi {
     font-size: 12px;
 }
+
+.cardTitleTitle {
+    flex: 1;
+}
+.cardTitlePrivate {
+    flex: 1;
+    text-align: right;
+}
+.artifactDetailPrivate {
+    font-size: 32px;
```

Comment:
What does it look like if we don't override this?

Reply:
It's a bit too small. 
<img width="620" alt="image" src="https://user-images.githubusercontent.com/9397092/166492440-6e5061be-19f2-48a3-8ec8-4c182a0e4a84.png">